### PR TITLE
[Added] System Design Primer Github RepoAdded] 

### LINKED ISSUE #1984: [Add]  System Design Primer Github Repo
### Site URL

https://github.com/donnemartin/system-design-primer

### Description of the above provided site

The System Design Primer stands out as one of the most widely embraced GitHub repositories for acquiring knowledge in the realm of system design

### Category

Backend

### Sub Category

system design

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [X] I am participating in Hacktoberfest 2023

### PR DESCRIPTION
**Pull request by khushminder270**
## Fixes Issue 
Closes #1984

### CODE REVIEW

**Review by Anmol-Baranwal — CHANGES_REQUESTED** [review]
@CBID2 @rupali-codes 

Someone else created this issue, and he made those changes in this PR when the issue was not even unlocked to get assigned.

image

@khushminder270 
Please do the changes that you have mentioned in the issue, and not regarding other issues.

File: database/languages/c-programming.json [hunk 1]
```diff
@@ -12,5 +12,12 @@
         "url": "https://www.freecodecamp.org/news/learn-c-programming-classic-book-dr-chuck/",
         "category": "languages",
         "subcategory": "c-programming"
+    },
+    {
+        "name": "C Language Tutorial for Beginners",
+        "description": "C Language Tutorial for Beginners (with Notes & Practice Questions)",
```
Comment (CBID2) [inline database/languages/c-programming.json:18]:
```suggestion
        "description": "This is a C Language tutorial for beginners. It contains notes for reference & questions for practice",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @khushminder270. Your PR is off to a great start. I just gave some suggestions on how to improve it.

**Comment by Anmol-Baranwal** [comment]
@CBID2 
He needs to completely remove the resource for C programming, so there is no changes for the need.

**Comment by CBID2** [comment]
> @CBID2 
> 
> He needs to completely remove the resource for C programming, so there is no need for the changes.

Ahh ok @Anmol-Baranwal

**Comment by khushminder270** [comment]
@Anmol-Baranwal @CBID2 
Removed the changes related to other issues

**Comment by khushminder270** [comment]
@Anmol-Baranwal @CBID2
Please add **hacktoberfest-accepted** label to the PR

**Review by Anmol-Baranwal — APPROVED** [review]
🚀 Looks good to me.

**Comment by Anmol-Baranwal** [comment]
@CBID2 @rupali-codes 
He raised a PR for a issue which isn't assigned to him.

I'm closing this.

**Comment by CBID2** [comment]
> @CBID2 @rupali-codes 
> 
> He raised a PR for a issue which isn't assigned to him.
> 
> 
> 
> I'm closing this.

Sorry @Anmol-Baranwal. My mistake. @khushminder270, I suggest raising an issue.

**Comment by Anmol-Baranwal** [comment]
I noticed it, when the original author raised a PR. I even approved this.

**Comment by khushminder270** [comment]
@Anmol-Baranwal @CBID2 
I am not getting why it is marked as SPAM?

**Comment by CBID2** [comment]
> @Anmol-Baranwal @CBID2 
> 
> I am not getting why it is marked as SPAM?

Sorry about that

**Comment by Anmol-Baranwal** [comment]
> @Anmol-Baranwal @CBID2 I am not getting why it is marked as SPAM?

I would have seriously marked this as spam. It is always recommended to go with the flow.

Create a issue, then let us assign it and make a PR.

You created a PR for a issue which was assigned to someone else, believe me 99% maintainers would mark it as spam.

@CBID2 is kind enough to remove it. Just make sure to do it in the future.

Let us know if you want to know, how to contribute properly.

**Pull request closed without merging**